src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -191,6 +188,9 @@
     <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.LookupAccountNameW.cs">
       <Link>Common\Interop\Windows\mincore\Interop.LookupAccountNameW.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.SHGetKnownFolderPath.cs">
```

Comment:
They should be kept separate - one is meant for classic WIndows apps and other is for UWP apps. There are other bugs in the authoring of the .csproj file - but merging them would be going in a wrong direction.